### PR TITLE
Items component

### PR DESCRIPTION
**Pull request by dmarcautan**

### CODE REVIEW

File: src/components/Content/Items/Items.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+import { Well } from "react-bootstrap";
+
+const joinClasses = (class1String, class2String) => (class1String || "") + (class2String ? " " + class2String : "");
```
Comment (Kashkovsky) [inline src/components/Content/Items/Items.js]: Use classnames for this stuff import classNames from "classnames"

File: src/index.css [hunk 2]
```diff
@@ -510,4 +510,61 @@ Notes on possible css class implementation:
 	padding-left: 0.75rem;
 }
 
-/** DateTimePicker **/
+/** Items **/
```
Comment (Kashkovsky) [inline src/index.css]: Consider creating a separate css file which can be imported in elements styleImports

File: src/components/Content/Items/Items.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+export default function Items(props) {
+	const { items, renderItem, onItemRemove, containerClassName, itemClassName } = props;
+	return (
+		<Well bsSize="sm" className={joinClasses("eui-items", containerClassName)}>
+			<ul className="eui-items__list">
+				{items.map((item, index) => (
+					<li key={index} className={joinClasses("eui-items__list__item", itemClassName)}>
```
Comment (Kashkovsky) [inline src/components/Content/Items/Items.js]: Use some more unique value for key prop

File: src/components/Content/Items/Items.js (hunk 1 above)
Comment (Kashkovsky) [reply src/components/Content/Items/Items.js]: `classNames("eui-items", containerClassName)` will do the job

File: src/index.css (hunk 2 above)
Comment (Kashkovsky) [reply src/index.css]: Agree

File: src/components/Content/Items/Items.js (hunk 3 above)
Comment (Kashkovsky) [reply src/components/Content/Items/Items.js]: Agree

File: src/components/Content/Items/Items.js (hunk 1 above)
Comment (dmarcautan) [reply src/components/Content/Items/Items.js]: Right, it works. Fixed

**Pull request merged**